src/main/java/org/terasology/signalling/blockFamily/SignalCableBlockFamilyFactory.java

```diff
@@ -137,7 +144,14 @@ public BlockFamily createBlockFamily(BlockFamilyDefinition definition, BlockBuil
         return new SignalUpdateFamily(blockUri, definition.getCategories(),
                 archetypeBlock, blocksForConnections, (byte)63);
     }
-    
+
+    /**
+     * Links the given blocks together.
+     * <p>
+     * @param basicBlocks An array of blocks to
+     * @param index The index of the block to put a shape mapping
+     * @param connections
+     */
```

Comment:
Seems like you've dropped a word on the `basicBlocks` param and the `connections` param is blank.

For `connections` something like "The connection type being added" would be fine